File: C++/condicionales if else.cpp
#include <iostream>
using namespace std;

int main() {
    int a = 2;

    // Condición simple
    if (a == 2) {
        cout << "a vale 2" << endl;
    } else {
        cout << "a es diferente de 2" << endl;
    }

    // Concatenación de condiciones
    string Nombre = "Santiago";
    int Edad = 23;
    string Pais = "Colombia";

    if (Nombre == "Santiago" && Edad == 23 && Pais == "Colombia") {
        cout << "Su nombre es " << Nombre << ", tiene " << Edad << " y es de " << Pais << endl;
    } else if (Nombre == "Santiago" && Edad != 23) {
        cout << "Su nombre es Santiago y no tiene 23 años" << endl;
    } else if (Nombre != "Santiago" && Edad == 23) {
        cout << "Su nombre no es Santiago y tiene 23 años" << endl;
    } else {
        cout << "No se llama Santiago ni tiene 23 años" << endl;
    }

    return 0;
}

File: JAVA/condicionales if else.java
public class Condiciones {
    public static void main(String[] args) {
        int a = 2;
        
        // Condición simple
        if (a == 2) {
            System.out.println("a vale 2");
        } else {
            System.out.println("a es diferente de 2");
        }

        // Declaración de variables
        String nombre = "Santiago";
        int edad = 23;
        String pais = "Colombia";

        // Evaluación de múltiples condiciones
        if (nombre.equals("Santiago") && edad == 23 && pais.equals("Colombia")) {
            System.out.println("Su nombre es " + nombre + ", tiene " + edad + " y es de " + pais);
        } else if (nombre.equals("Santiago") && edad != 23) {
            System.out.println("Su nombre es Santiago y no tiene 23 años");
        } else if (!nombre.equals("Santiago") && edad == 23) {
            System.out.println("Su nombre no es Santiago y tiene 23 años");
        } else {
            System.out.println("No se llama Santiago ni tiene 23 años");
        }
    }
}
